redash/query_runner/presto.py

```diff
@@ -98,6 +98,7 @@ def run_query(self, query, user):
             port=self.configuration.get("port", 8080),
             protocol=self.configuration.get("protocol", "http"),
             username=self.configuration.get("username", "redash"),
+            source='redash',
```

Comment:
Could we make this configurable in the data source setup screen? `redash` by default but the user could change it?

Reply:
Thats a valid request. Let me pull that line from this PR.